static/docs/user-guide/howto/external-data-and-cache.md

```diff
@@ -0,0 +1,151 @@
+# Keep Data and Cache Outside the Project
+
+Sometimes we would like to keep the data and cache outside the project
```

Comment:
not sure how is it related to being able to search code tbh, could you elaborate?